girder/models/model_base.py

```diff
@@ -61,8 +63,20 @@ def __init__(self):
         }
 
         self.initialize()
+        self._validateSchema()
         self.reconnect()
 
+    def _validateSchema(self):
+        """
+        Validate schema if one has been provided.
+        """
+        if self.schema:
+            try:
+                validator = jsonschema.validators.validator_for(self.schema)
+                validator.check_schema(self.schema)
+            except jsonschema.exceptions.SchemaError as se:
+                raise Exception('Invalid JSON schema: %s' % se.message)
```

Comment:
Maybe you could add a bit of debugging here, like the name of the model at least.

I've had a hard time hunting down these json schema validation errors in the past.